workflow/rules/trimming.smk

```diff
@@ -55,7 +55,7 @@ rule merge_fastqs:
     input:
         lambda w: expand("trimmed/{{sample}}-{unit}.{{read}}.fastq.gz", unit=units.loc[w.sample, "unit_name"])
     output:
-        "merged/{sample}.{read,(1|2)}.fastq.gz"
+        "merged/{sample}.{read,(single|1|2)}.fastq.gz"
```

Comment:
Actually, the conditional on whether having gz or not is not necessary. gzipped files can be catted like normal files. Further, since both output and input are always gz there wouldn't even be the need to distinguish if it was necessary.